examples/mplot3d/contour3d_3.py

```diff
@@ -1,30 +1,31 @@
 """
-========================================
-Projecting contour profiles onto a graph
-========================================
+=====================================
+Project contour profiles onto a graph
+=====================================
 
 Demonstrates displaying a 3D surface while also projecting contour 'profiles'
 onto the 'walls' of the graph.
 
-See contourf3d_demo2 for the filled version.
+See :doc:`contourf3d_2` for the filled version.
 """
 
 from mpl_toolkits.mplot3d import axes3d
 import matplotlib.pyplot as plt
-from matplotlib import cm
+from matplotlib.cm import coolwarm as cmap
```

Comment:
It's recommended nowadays to use the colormap registry instead of importing from `matplotlib.cm`:

```
cmap = plt.colormaps['coolwarm']
``` 
You could go even one step further and use the string `contour(..., cmap='coolwarm')`. Usually, that's the simplest way if you don't need the colormap object itself. Here however, you have the `cmap(0)` call. I'm tempted to resolve this to an explicit `'#3b4cc0'` and get rid of the colormap object completely, which IMHO makes the example simpler.

But I'll let you choose whether to go all-string or retrieve `cmap` from `plt.colormaps`.

Reply:
Yes, that `cmap(0)` is bugging me too. I was considering `'#3b4cc0'` too, but it is too cryptic. Is there some named blue in the neighborhood?  

https://matplotlib.org/stable/gallery/color/named_colors.html#css-colors

Since I am green blind, I would like to kindly ask you to please help me out here: If you agree, please suggest a named blue instead of `'#3b4cc0'`. Thanks!

Reply:
![image](https://user-images.githubusercontent.com/2836374/191126393-3ac646e9-59d8-4067-9d7e-3e010b638bd1.png)

You can try 'mediumblue' which is slightly darker, or 'royalblue' which is a bit lighter.